pcre2/Dockerfile

```diff
@@ -1,21 +0,0 @@
-# Copyright 2016 Google Inc.
```

Comment:
I don't think you intend to remove pcre2 stuff. I think you probably should start of a new branch, right now it shows too many older commits. or just squash all the commits into one.